test/test_tmpltbank.py

```diff
@@ -196,13 +197,19 @@ def setUp(self):
         self.xis = vals
 
     def test_eigen_directions(self):
-        evalsStock = Array(numpy.loadtxt('%sstockEvals.dat'%(self.dataDir)))
-        evecsStock = Array(numpy.loadtxt('%sstockEvecs.dat'%(self.dataDir)))
+        fname='stockEvals.dat'
+        apy_fname = download_file(DATA_FILE_URL.format(fname), cache=False)
+        evalsStock = Array(numpy.loadtxt(apy_fname))
+
+        fname='stockEvecs.dat'
+        apy_fname = download_file(DATA_FILE_URL.format(fname), cache=False)
+        evecsStock = Array(numpy.loadtxt(apy_fname))
+
         maxEval = max(evalsStock)
         evalsCurr = Array(self.metricParams.evals[self.f_upper])
         evecsCurr = Array(self.metricParams.evecs[self.f_upper])
-        numpy.savetxt('newEvals.dat', evalsCurr)
-        numpy.savetxt('newEvecs.dat', evecsCurr)
+        #numpy.savetxt('newEvals.dat', evalsCurr)
+        #numpy.savetxt('newEvecs.dat', evecsCurr)
```

Comment:
I would just remove these lines rather than commenting them (same comment below).

Reply:
... I note that they should never have been uncommented in the version committed to the main branch. I should have commented them out before uploading the code in the first place.